WalletWasabi.Gui/Icons/Icons.xaml

```diff
@@ -142,7 +147,7 @@
       <!-- LoadWalletViewModel Icons -->
       <DrawingGroup x:Key="LoadWallet_Folder">
         <DrawingGroup.Children>
-          <GeometryDrawing Brush="#FFF6F6F6" Geometry="F1 M 19,50L 28,34L 63,34L 54,50L 19,50 Z M 19,28.0001L 35,28C 36,25 37.4999,24.0001 37.4999,24.0001L 48.75,24C 49.3023,24 50,24.6977 50,25.25L 50,28L 53.9999,28.0001L 53.9999,32L 27,32L 19,46.4L 19,28.0001 Z " />
+          <GeometryDrawing Brush="#FFB3B3B3" Geometry="F1 M 19,50L 28,34L 63,34L 54,50L 19,50 Z M 19,28.0001L 35,28C 36,25 37.4999,24.0001 37.4999,24.0001L 48.75,24C 49.3023,24 50,24.6977 50,25.25L 50,28L 53.9999,28.0001L 53.9999,32L 27,32L 19,46.4L 19,28.0001 Z " />
```

Comment:
Is this supposed to be here?

Reply:
No, it should not. Thanks for reviewing.